tests/functionnal/Telemetry.php

```diff
@@ -41,20 +41,21 @@ class Telemetry extends DbTestCase {
    public function testGrabGlpiInfos() {
       $expected = [
          'uuid'               => 'TO BE SET',
+         'type'               => 'install',
          'version'            => GLPI_VERSION,
          'plugins'            => [],
          'default_language'   => 'en_GB',
          'install_mode'       => GLPI_INSTALL_MODE,
          'usage'              => [
-            'avg_entities'          => '0-500',
-            'avg_computers'         => '0-500',
-            'avg_networkequipments' => '0-500',
-            'avg_tickets'           => '0-500',
-            'avg_problems'          => '0-500',
-            'avg_changes'           => '0-500',
-            'avg_projects'          => '0-500',
-            'avg_users'             => '0-500',
-            'avg_groups'            => '0-500',
+            'avg_entities'          => '1-500',
+            'avg_computers'         => '1-500',
+            'avg_networkequipments' => '0',
```

Comment:
I disagree. All average must be an average, not a single value.

Reply:
@trasher As exposed on the ticket, I suggest to be able to detect if a functionnality isn't used or just have a low usage.
Imho, this information could help developpers to known which part of the product is (really) used.

Regards,
Anael

Reply:
We choose to be vague by design; I' really not sure about this change. Will wait for @orthagh point of view.

Reply:
i'm not opposed to the change, informing of no usage don't disclose that much, i think.
moreover, we don't exploit yet this part of telemtry in frontend website, so meh !